content/_data/changelogs/lts.yml

```diff
@@ -4785,6 +4785,328 @@
     message: |-
       Don't tell user's to signup when signup is not available.
 
+- version: "2.277.1"
+  date: 2021-03-10
+  lts_predecessor: "2.263.4"
+  lts_baseline: "2.277"
+  banner: >
+    As described in the <a href="/blog/2020/11/10/major-changes-in-weekly-releases/">Major changes in the weekly release line</a> blog post, this release improves key user interface components.
+    Configuration user interfaces now use html <code>div</code> markup to present a better layout than the previous html <code>table</code> layout.
+    Configuration pages are now easier to read, easier to understand, and work better on a wide range of screens.
+    The jQuery library used inside Jenkins has been upgraded from 2.1.4 to 3.5.1.
+    <p>&nbsp;</p>
+    <p>As described in the <a href="/blog/2020/11/10/spring-xstream/">Spring and XStream updates (breaking changes!)</a> blog post, this release updates and replaces outdated internal components.
+    The Acegi security library used for authentication has been replaced by Spring Security (<a href="https://github.com/jenkinsci/jep/blob/master/jep/227/README.adoc">JEP-227</a>).
+    A fork of the XStream library used to read and write XML files has been replaced by the upstream version of XStream (<a href="https://github.com/jenkinsci/jep/blob/master/jep/228/README.adoc">JEP-228</a>).
+
+  changes: # compared to lts_baseline 2.277 - extracted from the RC commit(s)
+
+  - type: major bug
+    category: major bug
+    pull: 5282
+    issue: 41727
+    authors:
+    - timja
+    message: |-
+      Show available plugin updates by reloading update center data on upgrade/downgrade.
+
+  # Regressions introduced after 2.263.4 and resolved before 2.277.1 was released
+  #
+  # pull: 5216 (PR title: [JENKINS-64746] public API hudson.model.UpdateCenter.HudsonUpgradeJob constructor signature change)
+  # pull: 5236 (PR title: [JENKINS-64775] Restores plugin details in DOM)
+  # pull: 5263 (PR title: [JENKINS-64805] Fixes IE11 error on the available tab)
+  # pull: 5281 (PR title: [JENKINS-64840] Fix searching multiple update sites)
+  # pull: 5285 (PR title: Bump spring-security-bom from 5.4.2 to 5.4.4)
```

Comment:
Please note all of these. If it's important enough to backport, it's important enough to note, even if it doesn't matter to users of the previous LTS line. They have their separate section with highlights below.

Most notably, the last one in this list fixes a security vulnerability present in 2.277.

If any changes originally went without a changelog entry, there's a disconnect between backport and changelog authors regarding importance of changes (why would we risk regressions for something so irrelevant it doesn't even get a changelog entry?), for the reason I just mentioned.

Reply:
Done in 18b570db12cccb184ff0ba3188e732f1ae81ac29